tests/unit/_utils/test_timedelata_ms.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+from datetime import timedelta
+from typing import Any
+
+import pytest
+from pydantic import BaseModel
+
+from crawlee._utils.models import timedelta_ms
+
+
+class _ModelWithTimedeltaMs(BaseModel):
+    time_delta: timedelta_ms | None = None
+
+
+@pytest.mark.parametrize(
+    ('time_delta_input', 'expected_time_delta', 'expected_model_dump_value'),
+    [
+        (1.0, timedelta(milliseconds=1), 1),
+        (1, timedelta(milliseconds=1), 1),
+        ('1', timedelta(milliseconds=1), 1),
+        (timedelta(milliseconds=1), timedelta(milliseconds=1), 1),
+        (3.01, timedelta(microseconds=3010), 3),
+        (3.5, timedelta(microseconds=3500), 4),
+        (3.99, timedelta(microseconds=3990), 4),
+        (None, None, None),
+        (float('inf'), timedelta(days=999999999, seconds=3600 * 24 - 1, microseconds=999999), float('inf')),
```

Comment:
I considered 3600 * 24 - 1 to be more readable than 86399. Since it is just a test, I avoided defining SECONDS_IN_HOUR and HOURS_IN_DAYS ....

Reply:
Thanks for avoiding that :slightly_smiling_face: I usually do `timedelta(days=3, hours=8).total_seconds()` instead.